src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -156,6 +165,7 @@ const CourtCaseListEntry: React.FC<Props> = ({ courtCase, currentUser }: Props)
           ) : (
             <LockedByTag lockedBy={errorLockedByUsername} />
           )}
+          {<CaseUnlockedTag isCaseUnlocked={exceptionHasBeenRecentlyUnlocked && triggerHasBeenRecentlyUnlocked} />}
```

Comment:
I think we'd need check whether the case actually unlocked, something like:
```suggestion
          {<CaseUnlockedTag isCaseUnlocked={exceptionHasBeenRecentlyUnlocked && triggerHasBeenRecentlyUnlocked && !errorLockedByUsername} />}
```

Otherwise users may see something like this:
![image](https://user-images.githubusercontent.com/22743709/224715923-b7165718-241e-4896-b3af-4f9c4cab5407.png)